ast/parser.go

```diff
@@ -105,8 +114,8 @@ type ParserOptions struct {
 	FutureKeywords     []string
 	SkipRules          bool
 	JSONOptions        *astJSON.Options
+	RegoVersion        RegoVersion
```

Comment:
This is a breaking change. We could also keep the old `RegoV1Compatible` parameter, but I think that'll just be confusing. It's unlikely anyone is using this yet, so I feel this isn't a huge transgression.

Reply:
Should we deprecate the old one and remove it in a future release? I agree not many must be using this.

Reply:
I've added it back and added a deprecation notice.